ui/tests/acceptance/pageobjects/accountsPage.js

```diff
@@ -40,6 +40,11 @@ module.exports = {
         util.format(this.elements.currentRole.selector, role)
 
       return this.useXpath().expect.element(roleSelector).to.be.visible
+    },
+
+    navigateToAccounts: async function () {
+      await this.waitForElementVisible('@loadingAccountsList')
+      return this.useXpath().expect.element('@accountsListTable').to.not.present
```

Comment:
Fix the function name. also, we generally try not to do assertions on pageObjects.

Reply:
Fixed